lib/Extension/Core/Model/JsonSchemaBuilder.php

```diff
@@ -53,10 +53,12 @@ public function dump(): string
                 $meta = [
                     'description' => $definition->description(),
                 ];
-                if ($definition->types()) {
-                    $meta['type'] = $this->mapTypes($definition->types());
+
+                $mappedTypes = $this->mapTypes($definition->types());
+                if ($mappedTypes) {
+                    $meta['type'] = $mappedTypes;
                 }
-                if ($definition->defaultValue()) {
+                if ($definition->defaultValue() || in_array('boolean', $mappedTypes)) {
```

Comment:
sorry to miss this PR, should this rather be `null !== $definition->defaultValue()` ? tbh there is a design flaw here as `null` could also be a default value, but this would be OK for now.

Reply:
I also forgot about it but I added a test now for it as well. One day if all extensions are documented this check isn't needed anymore.